lib/emque/consuming/adapters/rabbit_mq/manager.rb

```diff
@@ -74,6 +90,18 @@ def initialize_workers
             }
           end
 
+          def initialize_delayed_message_workers
+            self.delayed_message_workers = [].tap { |workers|
```

Comment:
what is the advantage of tap + a times loop vs a single Array.new(size) loop? I find it slightly harder to parse with the 2 constructs here.